### ISSUE TITLE
0.3 ptyr questions

### ISSUE THREAD

**Issue by alexbjorling** [ptycho/ptypy#185]
Hi folks,

Upgrading to 0.3 at the beamline (where we had a master snapshot from 2016...). Two questions so far:

* The `ptyr` files (which we read for propagating when aligning our KB focus) look different. Probe data used to be in `content/probe/S00G00/data` but is now in `content/probe/SNMG00/data`. Bug or feature?

* I get a lot of validator warnings which makes people nervous. Is it supposed to look like this on every autosaving iteration?
```
---------------------------------- Autosaving ----------------------------------
WARNING root - Save file exists but will be overwritten (force_overwrite is True)
Generating copies of probe, object and parameters and runtime
ERROR root -                                                    verbose_level        INVALID
ERROR root -                                                    engines              INVALID
ERROR root -                                                    data_type            INVALID
ERROR root -                                                    dry_run              INVALID
ERROR root -                                                    run                  INVALID
ERROR root -                                                    ipython_kernel       INVALID
ERROR root -                                                    scans                INVALID
ERROR root - io                                                 autoplot             INVALID
ERROR root - io                                                 home                 INVALID
ERROR root - io                                                 rfile                INVALID
ERROR root - io                                                 autosave             INVALID
ERROR root - io.interaction                                     active               INVALID
WARNING root - The parameters we are saving won't pass a validator check!
Saving to /home/alex/ptycho/ptypy3test/dumps/basic_reconstruction_script/basic_reconstruction_script_DM_0010.ptyr
--------------------------------------------------------------------------------
```

**Comment by bjoernenders** [comment]
Hi Alex, 
thanks for the report.
To 1 : I would call it a feature, the name is derived from the scan name of the input tree.
To 2 : I pushed a fix 75dcf81. There was a change in hierarchy of the parameters that the saving parameter test didn't account for. Funny that I didn't catch that bug. 
Best, Bjoern

**Comment by alexbjorling** [comment]
1: Haha I didn't realize `NM` was the name of my scan, I assumed it was a pair of counters `(n, m)` that hadn't been parsed.

2: Thanks. Very agile! It looks much nicer now.
